helloworld/util/BUILD

```diff
@@ -12,22 +13,21 @@ python_distribution(
   ),
 )
 
+# `name` defaults to the name of this directory, i.e., `util`.
+# `sources` defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
 python_library(
-    # `name` defaults to the name of this directory, i.e., `util`.
-    # `sources` defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
-    dependencies=[
-        "//:setuptools",
-        "//:translate",
-        "helloworld/util/proto",
-    ],
+  # Pants cannot infer a dependency on Protobuf, so we explicitly add it.
+  dependencies=[
+      "helloworld/util/proto",
+  ],
 )
 
 python_tests(
     name='tests',
     # `sources` defaults to ['*_test.py', 'test_*.py', 'conftest.py'].
+    # Pants cannot infer a dependency on JSON files, so we explicitly add it.
```

Comment:
Nit: Is it more correct to say "on `resources` targets"? If you put a Python file in a `resources` target we (correctly) wouldn't infer a dep on it either.

Reply:
It is. Good suggestion.